index.d.ts

```diff
@@ -48,9 +57,12 @@ openEditor([
 openEditor([
 	'unicorn.js:5:3',
 ]);
+
+await openEditor(['unicorn.js:5:3'], {wait: true});
+console.log('File was closed');
```

Comment:
This should instead be an example in the docs for the `wait` option, like in the readme.